src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/ReportedOffence.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.entities
+
+import javax.persistence.Entity
+import javax.persistence.Table
+
+@Entity
+@Table(name = "reported_offence")
+data class ReportedOffence(
+  override val id: Long? = null,
+  var offenceCode: Int,
+  var victimPrisonersNumber: String? = null,
```

Comment:
Will we need this to expand later to include staff victims and collaborators?

Reply:
Yes indeed